sklearn/tests/test_naive_bayes.py

```diff
@@ -193,6 +194,15 @@ def test_gnb_naive_bayes_scale_invariance():
     assert_array_equal(labels[1], labels[2])
 
 
+# TODO: Remove in version 0.26
+@pytest.mark.parametrize("cls", [MultinomialNB, BernoulliNB, CategoricalNB])
+def test_discretenb_deprecated_coef_intercept(cls):
+    est = cls().fit(X2, y2)
+
+    for att in ["coef_", "intercept_"]:
+        assert_warns(FutureWarning, hasattr, est, att)
+
+
 @pytest.mark.parametrize("cls", [MultinomialNB, BernoulliNB, CategoricalNB])
 def test_discretenb_prior(cls):
```

Comment:
`ignore_warnings` and add comment to be removed in 0.26

Reply:
> `ignore_warnings` and add comment to be removed in 0.26

Should be ignored or tested that `FutureWarning` is raised with `pytest.warns`?

Reply:
We already test for that it warns in `test_discretenb_deprecated_coef_intercept`. All other tests that use them, we normally ignore them and comment to remove when the attribute gets deprecated.

Reply:
Thanks @thomasjpfan!

I think that I have wrapped by the `ignore_warnings` decorator (and a comment to remove) all testing methods where `coef_` and `intercept_` appear.